db/sql/10_msar_joinable_tables.sql

```diff
@@ -29,7 +29,7 @@ whether to travel from referrer to referant (when False) or from referant to ref
 */
 
 
-CREATE TYPE mathesar_types.joinable_tables AS (
```

Comment:
I like that you're moving the `joinable_tables` type out of the `mathesar_types` schema, but I'd expect it to go in `msar` instead of `__msar`.

Reply:
Done!

Reply:
Thanks 🙂